pkg/chains/formats/slsa/internal/material/material_test.go

```diff
@@ -400,6 +400,7 @@ func TestStructuredResultPipelineMaterials(t *testing.T) {
 	}
 }
 
+/*
```

Comment:
Let's either update the tests or remove them.

Reply:
Done